app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name,
+            health=100,
+            hidden=False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def die(self) -> None:
+        Animal.alive.remove(self)
+
+    def __str__(self) -> str:
+        return f"{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}"
```

Comment:
Your string formatting is not correct, it should be 
```suggestion
        return f"{{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}}"
```
You could add line breaking inside of `()` to improve the readability,

Reply:
Why should I use here double "{"?